src/bindings/python/wheel/requirements-dev.txt

```diff
@@ -1,4 +1,4 @@
 -c ../constraints.txt
```

Comment:
should we limit it in the constraints.txt file instead?

Reply:
Added a proper fix w/o limiting `wheel` version